react/features/lobby/middleware.ts

```diff
@@ -13,6 +15,11 @@ import StateListenerRegistry from '../base/redux/StateListenerRegistry';
 import { playSound, registerSound, unregisterSound } from '../base/sounds/actions';
 import { isTestModeEnabled } from '../base/testing/functions';
 import { handleLobbyChatInitialized, removeLobbyChatParticipant } from '../chat/actions.any';
+import { navigate }
```

Comment:
This is going to break web. We can't import platform dependent code in a platform agnostic middleware...